.github/workflows/linux.yml

```diff
@@ -823,3 +810,95 @@ jobs:
             ${{ env.INSTALL_TEST_DIR }}/logs/interapted/*.log
             ${{ env.INSTALL_TEST_DIR }}/logs/disabled_tests.log
           if-no-files-found: 'error'
+
+  TensorFlow_Hub_Models_Tests:
+    needs: Build
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04
+    env:
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
+
+    steps:
+      - name: Create Directories
+        run: |
+          mkdir -p ${{ env.INSTALL_DIR }} ${{ env.INSTALL_TEST_DIR }}
+
+      - name: Clone OpenVINO
+        uses: actions/checkout@v4
+        with:
+          path: 'openvino'
+          submodules: 'recursive'
+
+      #
+      # Dependencies
+      #
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install python dependencies
+        run: |
+          # For Python API
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/wheel/requirements-dev.txt
+          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/requirements.txt
```

Comment:
since we instead wheels, do we need any other dependencies?
and probably, we need to remove clone of OpenVINO as well

Reply:
Okay, let me get CI green first and I will try to remove redundant dependencies installation from steps.
Agree, I believe it is possible.

Reply:
Those files contains dependencies to:
- run `openvino` (but openvino wheel already contains numpy dependency)
- dependencies to build openvino wheel. They are not required at this stage.
So, we can safely remove those files as well as OpenVINO repo cloning

Reply:
Done